kitsune/sumo/static/sumo/js/instant_search.es6

```diff
@@ -245,4 +246,21 @@
 
     ev.preventDefault();
   });
+
+  function loadFromHistory(e) {
+    let state = e.type == "popstate" ? e.state : history.state;
+    if (state?.query) {
+      window.k.InstantSearchSettings.hideContent();
+      $('[data-instant-search="form"] input[name="q"]').each(function () {
+        $(this).val(state.query);
+      });
+      search.params = state.params;
+      search.query(state.query, k.InstantSearchSettings.render);
+    } else {
+      window.k.InstantSearchSettings.showContent();
```

Comment:
We should also clear the input field or make it respond to user input (by pressing enter or the search icon)

Reply:
The input was already being cleared in showContent for a couple of fields, but not all of them - it should be now.

Enter or search icon should work anyways, I've opened an issue for that: https://github.com/mozilla/sumo-project/issues/785